frontend/src/pages/notebookController/screens/server/GPUSelectField.tsx

```diff
@@ -57,10 +61,26 @@ const GPUSelectField: React.FC<GPUSelectFieldProps> = ({ value, setValue }) => {
     };
   }, [notification, areGpusAvailable]);
 
+  React.useEffect(() => {
```

Comment:
Why not make this part of the fetch logic? This hook only runs once per fetch call, no?